kubernetes/aws-kops/k8s.yaml

```diff
@@ -0,0 +1,399 @@
+                    - stripe-1
+                    - stripe-2
+              topologyKey: kubernetes.io/hostname
+      containers:
+        - name: terracotta
+          image: store/softwareag/terracotta-server:10.2
+          command: ["bin/start-tc-server.sh"]
+          args: ["-f", "/configs/stripe2.xml", "-n", "$(POD_NAME)"]
+          imagePullPolicy: Always
+          env:
+            - name: ACCEPT_EULA
+              value: "Y"
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          ports:
+            - containerPort: 9410
+              name: terracotta-port
+            - containerPort: 9430
+              name: sync-port
+          volumeMounts:
+            - name: config-volume
+              mountPath: /configs
+            - name: dataroots
+              mountPath: /data/dataroots
+            - name: backups
+              mountPath: /data/backups
+      imagePullSecrets:
+        - name: regcred
+      volumes:
+        - name: config-volume
+          configMap:
+            name: tc-configs
+  volumeClaimTemplates:
+    - metadata:
+        name: dataroots
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: 100Gi
+    - metadata:
+        name: backups
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: 100Gi
+
+---
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cluster-tool
+spec:
+  template:
+    metadata:
+      name: cluster-tool
+    spec:
+      initContainers:
+        - name: init-dns
+          image: busybox
+          command: ["sh", "-c", "hostnames=\"terracotta-1-0.stripe-1 terracotta-1-1.stripe-1 terracotta-2-0.stripe-2 terracotta-2-1.stripe-2\"; for hostname in $hostnames; do echo waiting for ${hostname}:9410; nc -z ${hostname} 9410; if [ $? -ne 0 ]; then exit 1; fi; done"]
+      containers:
+        - name: cluster-tool
+          image: store/softwareag/terracotta-cluster-tool:10.2
+          args: ["configure", "-n", "MyCluster", "/configs/stripe1.xml", "/configs/stripe2.xml"]
+          env:
+          - name: ACCEPT_EULA
+            value: "Y"
+          volumeMounts:
+          - name: config-volume
+            mountPath: /configs
+          - name: license-volume
+            mountPath: /licenses
+      imagePullSecrets:
+        - name: regcred
+      volumes:
+        - name: config-volume
+          configMap:
+            name: tc-configs
+        - name: license-volume
+          configMap:
+            name: license
+      restartPolicy: OnFailure
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: tmc
+spec:
+  ports:
+  - name: tmc-port
+    port: 9480
+  type: LoadBalancer
+  selector:
+    app: tmc
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: tmc-internal
+  labels:
+    app: tmc
+spec:
+  ports:
+  - name: http
+    port: 9889
```

Comment:
nope, no port 9889 in tcdb

Reply:
yup. missed to update the port. The deployment is not having any issue as headless service do not route the traffic.